tests/test_bidstream_client.py

```diff
@@ -112,25 +115,38 @@ def test_phone_uids(self):  # PhoneTest
                 self.assertEqual(result.identity_scope, expected_scope)
                 self.assertEqual(result.advertising_token_version, expected_version)
 
-    def test_token_lifetime_too_long_for_bidstream(self):  # TokenLifetimeTooLongForBidstream
-        expires_in_sec = IN_3_DAYS + dt.timedelta(minutes=1)
+    def test_token_lifetime_too_long_for_bidstream_but_remaining_lifetime_allowed(self):  # TokenLifetimeTooLongForBidstreamButRemainingLifetimeAllowed
+        generated = YESTERDAY
+        expires_in_sec = generated + dt.timedelta(days=3) + dt.timedelta(minutes=1)
```

Comment:
Can we extract a variable for max bidstream lifetime from here, line 134, and line 33 so we can see the connection